insights/parsers/nmcli.py

```diff
@@ -153,3 +156,38 @@ def connected_devices(self):
             if 'STATE' in self.data[key] and self.data[key]['STATE'] == 'connected':
                 con_dev.append(key)
         return con_dev
+
+
+@parser(Specs.nmcli_conn_show)
+class NmcliConnShow(CommandParser):
+    """
+    This file will parse the output of all the nmcli connections.
+
+    Sample configuration from a teamed interface in file ``/usr/bin/nmcli conn show``::
+
+       NAME      UUID                                  TYPE      DEVICE
+       enp0s3    320d4923-c410-4b22-b7e9-afc5f794eecc  ethernet  enp0s3
+       virbr0    7c7dec66-4a8c-4b49-834a-889194b3b83c  bridge    virbr0
+       test-net  f858b1cc-d149-4de0-93bc-b1826256847a  ethernet  --
+
+    Examples:
+        >>> type(static_conn)
+        <class 'insights.parsers.nmcli.NmcliConnShow'>
+        >>> static_conn.disconnected_connection
+        ['test-net-1']
+
+    Attributes:
+        data (list): list of connections wrapped in dictionaries
+        disconnected_connection (list): list of all the disconnected static route connections
```

Comment:
Since the docstring of the property `disconnected_connection` (line-192) have the same affect to this `Attributes`, better remove this line-181 to avoid duplicates

Reply:
You can check the documentation by executing the following command

~~~
# (cd docs; make html)
# Open the `.docs/_build/html/index.html` file in your browser and check its HTML page
~~~

Reply:
acknowledged, updated the code in the latest PR